spec/mergent/task_spec.rb

```diff
@@ -23,4 +23,12 @@
       expect(task.name).to eq "taskname"
     end
   end
+
+  describe "delegated methods" do
```

Comment:
I have no idea how to actually test this.

Reply:
I think what you've done here works. It's not elegant / pretty, but it does the job until we build out `Mergent::Task`.